crates/bevy_app/src/app_builder.rs

```diff
@@ -108,6 +117,8 @@ impl AppBuilder {
         self
     }
 
+    /// Adds a [`Stage`] with the given `label` to the last position of the
+    /// [startup schedule](CoreStage::Startup).
```

Comment:
While correct, i think linking to `CoreStage::Startup` leaks the actual implementation too much.

Reply:
Yes, your request actually makes sense, especially since `CoreStage` is just a collection of labels, not the stages themselves.

However, I wanted to keep the possibility to check up the role of those special stages. I thought of a couple solutions:

1. Add a description of the role of the various stages to `add_default_stages`, and link there instead.
2. Remove the link and add the role of the stages to the struct documentation.

I'm more for number 1, since 2 is not true in general (e.g. if `add_default_stages` isn't called). However we can specify that those stages are created in certain cases (already shown in the docs for `add_default_stages`).

Reply:
I opted for number 2.